catalog/opensearch/catalog-opensearch-endpoint/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -41,4 +41,18 @@
         <argument ref="filterBuilder"/>
     </bean>
 
+    <bean id="ddf.catalog.endpoint.opensearch"
```

Comment:
Hopefully the chance of them getting out of sync is small. The OpenSearchEndpoint class could directly implement the CatalogEndpoint interface but i don't know that it would solve the sync problem. There is no enforcement since it is not required. Any endpoint that would like to advertise itself (ex show up in registry as a federation method) should register a CatalogEndpoint. Right now this is the best way we have of making this information available.